integration-tests/debugger/redact.spec.js

```diff
@@ -2,48 +2,47 @@
 
 const { assert } = require('chai')
 const { setup } = require('./utils')
+const { once } = require('node:events')
 
 // Default settings is tested in unit tests, so we only need to test the env vars here
 describe('Dynamic Instrumentation snapshot PII redaction', function () {
   describe('DD_DYNAMIC_INSTRUMENTATION_REDACTED_IDENTIFIERS=foo,bar', function () {
     const t = setup({ env: { DD_DYNAMIC_INSTRUMENTATION_REDACTED_IDENTIFIERS: 'foo,bar' } })
 
-    it('should respect DD_DYNAMIC_INSTRUMENTATION_REDACTED_IDENTIFIERS', function (done) {
+    it('should respect DD_DYNAMIC_INSTRUMENTATION_REDACTED_IDENTIFIERS', async function () {
       t.triggerBreakpoint()
 
-      t.agent.on('debugger-input', ({ payload: [{ 'debugger.snapshot': { captures } }] }) => {
-        const { locals } = captures.lines[t.breakpoint.line]
+      const promise = once(t.agent, 'debugger-input')
```

Comment:
Note: I personally like this way of writing tests. That way it's somewhat easier for me to follow the logic and the events are always definitely going to cause the error to fail in case of an error.